packages/core/admin/ee/server/services/__tests__/stages.test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+jest.mock('@strapi/strapi/lib/utils/ee', () => {
+  const eeModule = () => true;
+
+  Object.assign(eeModule, {
+    features: {
+      isEnabled() {
+        return true;
+      },
+      getEnabled() {
+        return ['review-workflows'];
+      },
+    },
+  });
+
+  return eeModule;
+});
+
+const stageFactory = require('../review-workflows/stages');
+const { STAGE_MODEL_UID } = require('../../constants/workflows');
+
+const stageMock = {
+  id: 1,
+  name: 'test',
+  workflow: 1,
+};
+
+const entityServiceMock = {
+  findOne: jest.fn(() => stageMock),
+  findMany: jest.fn(() => [stageMock]),
+};
+
+const strapiMock = {
+  entityService: entityServiceMock,
+};
+
+const stagesService = stageFactory({ strapi: strapiMock });
+
+describe('Review workflows - Stages service', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('find', () => {
+    test('Should call entityService with the right model UID and ID', async () => {
+      stagesService.find({ workflowId: 1 });
+
+      expect(entityServiceMock.findOne).not.toBeCalled();
+      expect(entityServiceMock.findMany).toBeCalled();
+      expect(entityServiceMock.findMany).toBeCalledWith(STAGE_MODEL_UID, {
+        filter: { workflow: 1 },
+      });
```

Comment:
I do not understand how is this working if the workflow Id is not passed correctly. 🤔

Reply:
I have added one more stage in the integration test to be used for findById controller.